proxy_agent/src/common/hyper_client.rs

```diff
@@ -475,6 +478,18 @@ pub fn full_body<T: Into<Bytes>>(chunk: T) -> BoxBody<Bytes, hyper::Error> {
         .boxed()
 }
 
+// check the request could skip the sig
+// and stream the body to the server directly
+pub fn should_skip_sig(method: hyper::Method, relative_uri: hyper::Uri) -> bool {
```

Comment:
```suggestion
/// Certain endpoints are exempt from enforcement regardless of the VM's configuration.
/// Restricting access to these non-security impacting endpoints would introduce unreasonable
/// overhead and/or harm live-site investigations. Since the service won't require a signature,
/// there is no reason to generate one.
pub fn should_skip_sig(method: hyper::Method, relative_uri: hyper::Uri) -> bool {
```

This would make the context more clear imo

Reply:
Thanks, will update with your words.